pkg/tests/ossm/smcp_tls_ssl_test.go

```diff
@@ -43,16 +43,16 @@ func TestTLSVersionSMCP(t *testing.T) {
 		ns := "bookinfo"
 		t.Cleanup(func() {
 			oc.RecreateNamespace(t, ns)
-			oc.WaitSMCPReady(t, meshNamespace, smcpName)
+			oc.RecreateNamespace(t, meshNamespace)
 			oc.ApplyString(t, meshNamespace, smcpName)
```

Comment:
It's not needed inside the clean up, because the setup of each test will deploy a smcp